README.md

```diff
@@ -43,15 +43,33 @@ $ pip install -i https://test.pypi.org/simple/ pytweet
 from pytweet.pytweet import plot_timeline
 import pandas as pd
 
-plot_timeline(tweet_data, time_col)
+def helper_create_data():
+    """
+    Helper function for creating dataframe for testing
+    Parameters
+    -----------
+    Returns
+    --------
+    pandas.DataFrame
+        Returns a dataframe to be used for testing
+    Examples
+    ---------
+    >>> helper_create_data()
+    """
+    tweet_data = pd.read_csv("./tests/trumptweets-test.csv")
+    return tweet_data
+
+data = helper_create_data()
```

Comment:
```suggestion
tweet_data = pd.read_csv("./tests/trumptweets-test.csv")
plot_timeline(tweet_data, 'time')
```

Instead of including the helper, maybe it's better to use these two lines instead?

Reply:
Yes, this is a good suggestion.